scene/resources/curve.cpp

```diff
@@ -1683,7 +1683,7 @@ void Curve3D::_bake() const {
 			Vector3 forward = forward_ptr[idx];
 
 			Basis rotate;
-			rotate.rotate_to_align(frame_prev.get_column(2), forward);
+			rotate.rotate_to_align(-frame_prev.get_column(2), forward);
```

Comment:
Having tested it, I can assure that this is the wrong fix.

This prevents incorrect rotation, but it is equivalent to changing all `Basis::looking_at(forward, up)` to `Basis::looking_at(forward, up, true)`, which **again flips the forward direction**. You can see that the direction of PathFollow is reversed before and after this modification.